shell/client/admin/preinstalled-apps-client.js

```diff
@@ -13,8 +13,12 @@ Template.newAdminPreinstalledApps.onCreated(function () {
     state: "edit", // Other allowed states: "submitting", "success", and "error"
     message: undefined,
   });
-  const preinstalledApps = globalDb.getSettingWithFallback("preinstalledApps", {});
-  this.preinstalledAppIds = new ReactiveVar(_.pluck(preinstalledApps, "appId"));
+  const preinstalledApps = globalDb.getSettingWithFallback("preinstalledApps", []);
+  const preinstalledAppAndPackgeIds = {};
```

Comment:
Typo: `preinstalledAppAndPackgeIds` != `preinstalledAppAndPackageIds`

Not necessarily a showstopper, but it's a bit odd, so I thought I should mention it.